safe_transaction_service/history/views.py

```diff
@@ -231,10 +231,19 @@ def get_parameters(self) -> Tuple[bool, bool, bool]:
         )
         return executed, queued, trusted
 
-    def list(self, request, *args, **kwargs):
+    def get_tx_identifiers(self, request, *args, **kwargs) -> Optional[Response]:
```

Comment:
I think that `get_tx_identifiers` and `get_all_tx_identifiers` are too similar, coult work for you call this function `get_page_tx_indentifiers`?